rwengine/src/objects/PickupObject.hpp

```diff
@@ -194,7 +229,134 @@ class MoneyPickup : public PickupObject {
         money = m;
     };
 
-    bool onCharacterTouch(CharacterObject* character) override;
+    bool onPlayerTouch() override;
+};
+
+const static std::array<glm::vec3, 106> bigNVeinyPickupsLocations = {
```

Comment:
Can it be constexpr? It should be doable on almost every distro. (GLM 0.9.8.0 - 2016-09-11)